src/commands/init.rs

```diff
@@ -10,11 +10,8 @@ pub fn run(_namespace: Option<String>) {
         .join("cloup");
 
     if let Err(e) = fs::create_dir(&config_dirname) {
-        match e.kind() {
-            ErrorKind::PermissionDenied => {
-                eprintln!("Permission denied when creating config directory")
-            }
-            _ => (),
+        if e.kind() == ErrorKind::PermissionDenied {
```

Comment:
Probably worth keeping the match statement for if other error types want to be handled in the future.

Reply:
Got it. I've re-added this for upcoming commit